pkg/obs/controllerserver.go

```diff
@@ -82,7 +82,7 @@
 	credentials := cs.Driver.cloud
 	if tags, err := services.ListBucketTags(credentials, volName); err != nil {
 		return nil, err
-	} else if !isCreatByCSi(tags, "csi") {
+	} else if !isCreatedByCSi(tags, "csi") {
```

Comment:
I think the `csi` parameter is redundant. Because whether the bucket is created by CSI is determined internally by the function `isCreatedByCSi`, and there is no need to pass additional parameters externally

Reply:
done